common/helpers/field/index.js

```diff
@@ -30,6 +37,10 @@ function mapReferenceDataToOption({ id, title, key, conditional, hint }) {
     option.key = key
   }
 
+  if (checked) {
```

Comment:
I don't think we need to extend this helper for this functionality. This isn't something related to the item coming back from the reference data service, this is you needing to default an item to checked in a specific controller.

I would move that logic outside of this helper and keep it only where you need it.

Reply:
I think it's no really different from `hint` or `conditional` in the strict sense that we map attributes from a configuration object to the dom.